src/zarr/api/synchronous.py

```diff
@@ -675,6 +700,10 @@ def create(
     )
 
 
+def create_array(*args: Any, **kwargs: Any) -> Array:
+    return Array(sync(zarr.core.array.create_array(*args, **kwargs)))
```

Comment:
Please add a docstring for this function.

Reply:
I added one in #2607